install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -65,12 +65,12 @@ spec:
           - --controlPlaneAuthPolicy
           - MUTUAL_TLS
           - --discoveryAddress
-          - istio-pilot:15005
+          - istio-pilot.istio-system:15005
```

Comment:
Do we have a setting for istio namespace ? I believe some people want to install in other ns.

Reply:
Yes, I can refine this in a future PR... We have been hard coding namespace for pilot in a few other place too... we don't currently support running pilot in non istio-system ns officially.